product_variant_configurator/models/product_template.py

```diff
@@ -98,3 +98,12 @@ def name_search(self, name="", args=None, operator="ilike", limit=100):
                 if limit and len(res) >= limit:
                     break
         return res
+
+    def _get_variant_for_combination(self, combination):
+        """
+        Overwrite method to allow attributes that no create variants be selectable
+        """
+        self.ensure_one()
+        return self.env["product.product"].browse(
```

Comment:
This overwriting shouldn't be absolute, but only activated with a flag or something, but calling super in the rest of the cases.

Reply:
@pedrobaeza I will take a look later on